internal/service/advancedclustertpf/model_ClusterDescription20240805.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/diag"
```

Comment:
nit: in terms of file name can we fix string case? maybe `model_cluster` and `model_advanced_config` for now?

Reply:
The advantage of having them identical to SDK model is the simplified auto-generation and using a single admin version per model_xxx file.
Added it as a point to: [the next Jira task](https://jira.mongodb.org/browse/CLOUDP-286479)